config/datocms/migrations/1733671970_actionBlock.ts

```diff
@@ -0,0 +1,267 @@
+import { Client } from '@datocms/cli/lib/cma-client-node';
+
+export default async function (client: Client) {
+  console.log('Create new models/block models');
+
+  console.log(
+    'Create block model "\uD83C\uDF9B\uFE0F Action Block" (`action_block`)'
+  );
+  await client.itemTypes.create(
+    {
+      id: 'F60ZY1wFSW2qbvh99poj3g',
+      name: '\uD83C\uDF9B\uFE0F Action Block',
+      api_key: 'action_block',
+      modular_block: true,
+      inverse_relationships_enabled: false,
+    },
+    {
+      skip_menu_item_creation: true,
+      schema_menu_item_id: 'aV5wDaZcRLaLFiNqBpNstw',
+    }
+  );
+
+  console.log(
+    'Create block model "\uD83D\uDD17 Internal Link Block" (`internal_link_block`)'
+  );
+  await client.itemTypes.create(
+    {
+      id: 'GWnhoQDqQoGJj4-sQTVttw',
+      name: '\uD83D\uDD17 Internal Link Block',
+      api_key: 'internal_link_block',
+      modular_block: true,
+      inverse_relationships_enabled: false,
+    },
+    {
+      skip_menu_item_creation: true,
+      schema_menu_item_id: 'aVOBPryiQ8O5EiHEHDEang',
+    }
+  );
+
+  console.log('Creating new fields/fieldsets');
+
+  console.log(
```

Comment:
Should an ActionBlock be able to contain any number of actions (1+) with any style, like defined here? Or do we prefer a model where an ActionBlock has single modular fields for `primaryAction`, `secondaryAction` and `defaultAction` (all optional)?